Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/qrcode/VaccinationQRCodeExtractor.kt

```diff
@@ -56,6 +57,8 @@ class VaccinationQRCodeExtractor @Inject constructor(
             header = headerParser.parse(cbor),
             certificate = bodyParser.parse(cbor)
         ).also {
+            CertificateQrCodeCensor.addCertificateToCensor(it)
```

Comment:
Can we move this to the validator as well and remove the logging below? Why do we log the family name just to censor it?

Reply:
The validator is not used when parsing from storage, only the extractor. :thinking:

Reply:
Every QR code that enters the storage must have passed the validator and is captured there for censoring. Or am I misunderstanding how it is supposed to work? I do not want it in the extractor because it is used for repeated parsing.

Reply:
I guess it has to be like this. I was unaware of how the censor works.